prep_encoder.go

```diff
@@ -44,6 +44,10 @@ func (pe *prepEncoder) putArrayLength(in int) error {
 	return nil
 }
 
+func (pe *prepEncoder) putBool(in bool) {
+	pe.length++
```

Comment:
might be tempted to just delegate this to `putInt8` since that's the effective implementation elsewhere

Reply:
the above is mostly for symmetry :) bool is quite common in the new request/responses + just thinking it's nice to have `getBool()` sanity check for valid values. what do you think. is it ok to leave in?